README.md

```diff
@@ -86,3 +86,5 @@ Visit [Eiffel Community](https://eiffel-community.github.io) to get started and
    1. [Custom Data](./customization/custom-data.md)
 1. Extensions
    1. [Eiffel Operations Extension](https://github.com/Ericsson/eiffel-operations-extension)
+1. Maintenance
```

Comment:
Not sure Maintenance is the best choice of word but that's what I could come up with at this point.

Reply:
I'm not sure it's much better, but maybe more to the point anyway.
```suggestion
1. Releases and Milestones
```

Reply:
Yeah, that's probably better. Done.